src/RouterSelector.web.js

```diff
@@ -108,8 +109,13 @@ const RouterWrapper = () => {
       assign(tags, { isDeltaApp })
     }
 
-    initAnalytics(tags).then(() => log.debug('RouterSelector Rendered'))
-  }, [])
+    if (isLoggedInRouter && !analyticsInitializedRef.current) {
+      initAnalytics(tags).then(() => {
+        analyticsInitializedRef.current = true
+        log.debug('RouterSelector Rendered')
+      })
+    }
+  }, [isLoggedInRouter])
```

Comment:
This needs to fire whether the user is logged in or not.
Since this is an effect IT WILL fire once isLoggedInRouter is true. (so maybe it will fire twice)
If you really want this to only fire once you can check if isLoggedInRouter is first undefined and then it is either true/false, if so you can modify your if to `isLoggedInRouter != null` and remove the ref. otherwise you should remove this condition.

Reply:
@L03TJ3 try to add isBoolean() check instead of ref

Reply:
ah yes, logical error
Fixed now with isBoolean check